services/asset/properties/schemas.go

```diff
@@ -97,8 +25,9 @@ var propertySchema = map[string]*schema.Schema{
 	},
 	"node_id": {
 		Type:         schema.TypeString,
-		Required:     true,
+		Optional:     true,
```

Comment:
Wouldn't the users want to create nodes with properties ?

Reply:
yes I would prefer if they do it using the property resource instead of putting everything into the nodes
I think this was also the origin of this deprecation